usd/include/sdf/usd_parser/USDData.hh

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef SDF_USD_PARSER_USDDATA_HH
+#define SDF_USD_PARSER_USDDATA_HH
```

Comment:
As a follow-up to https://github.com/ignitionrobotics/sdformat/pull/817#discussion_r791011920, what if we changed all of the include guards in the USD -> SDF headers to have the `SDF_FROM_USD_<fileName>_HH_` format:
```suggestion
#ifndef SDF_FROM_USD_USDDATA_HH
#define SDF_FROM_USD_USDDATA_HH
```

Reply:
Include guards have been updated in https://github.com/ignitionrobotics/sdformat/pull/827/commits/2088f1e756d79cf5a7d1d304cc3a4c377ca28644